employees/views.py

```diff
@@ -497,10 +502,14 @@ class ExportReportsInProjectView(UserIsManagerOfCurrentProjectMixin, DetailView)
     model = Project
 
     def get_queryset(self) -> QuerySet:
-        return Project.objects.prefetch_related(
-            Prefetch(
-                "report_set",
-                queryset=Report.objects.filter(date__year=self.kwargs["year"], date__month=self.kwargs["month"]),
+        return (
+            super()
+            .get_queryset()
+            .prefetch_related(
```

Comment:
Please extract this to custom manager method and check if there are any usages around the app that could be replaced with it.